roles/zabbix_manage_repo/requirements.yml

```diff
@@ -0,0 +1,5 @@
+---
+- src: geerlingguy.apache
+- src: geerlingguy.mysql
+- src: geerlingguy.postgresql
+- src: community.postgresql
```

Comment:
Sorry, why do we need these `- src:` ?

Reply:
Never noticed it....It was actually in the server role which is what I copied when I created this.  Removing from both of them.